x-pack/plugins/security_solution/common/endpoint/models/policy_config.ts

```diff
@@ -12,6 +12,14 @@ import { PolicyConfig, ProtectionModes } from '../types';
 export const factory = (): PolicyConfig => {
   return {
     windows: {
+      advanced: {
```

Comment:
Why is this advanced section part of this PR? I think this probably belongs wiht the changes Jane is working on right? if so, I suggest it be removed from this PR